packages/common/src/store/pages/explore/selectors.ts

```diff
@@ -11,24 +11,16 @@ const getExplore = (state: CommonState) => state.pages.explore
 
 export const getExplorePlaylists = createSelector(
   (state: CommonState) => state.pages.explore.playlists,
-  getCollections,
-  getUsers,
-  (playlists, collections, users) => {
-    const explorePlaylists = playlists
-      .map((id) => collections[id])
-      .filter(Boolean)
-      .map((collection) => ({
-        ...collection,
-        user: users[collection.playlist_owner_id] || {}
-      }))
-    return explorePlaylists
-  }
+  (state: CommonState) => state.collections.entries,
```

Comment:
aren't we losing associated users here?

Reply:
So how does that affect the tile? It seems like the user is not referenced on the tile itself?

Reply:
oh you may be right. only thing i can think of is needing the handle for the click.